numpy/f2py/tests/test_crackfortran.py

```diff
@@ -88,6 +88,29 @@ def test_defaultPublic(self, tmp_path):
         assert 'private' not in mod['vars']['seta']['attrspec']
         assert 'public' in mod['vars']['seta']['attrspec']
 
+    def test_inlineDefinition(self, tmp_path):
```

Comment:
nit: it's unconventional naming of a method, suggest renaming the method to `test_access_type` or similar.

Reply:
Renamed as you suggested.